modules/nw-mutual-tls-auth.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * ingress/ingress-operator.adoc
+
+[id=nw-mutual-tls-auth_{context}]
+= Configuring Mutual TLS Authentication
+
+You can configure the Ingress Controller to enable mutual clientTLS(mTLS) authentication by setting `spec.clientTLS`. The clientTLS configures the Ingress controller to verify client certificates (CAs), certificate revocation list (CRL), and/or any certificate subject filters. Requests that do not provide valid certificates are rejected.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the cluster-admin role.
+* A ConfigMap that is in the openshift-config namespace.
+
+.Procedure
+
+Configure Mutual TLS authentication using the `spec.ClientTLS` field for the Ingress Controller.
+
+.  Edit the `IngressController` resource in the `openshift-ingress-operator` project:
+[source,terminal]
+----
+$ oc edit IngressController default -n openshift-ingress-operator
+----
+
+.  Edit the yaml:
+
+.Sample `IngressController` CR for a `clientTLS` profile that specifies filtering patterns.
+
+[source,yaml]
+----
+  apiVersion: operator.openshift.io/v1
+  kind: IngressController
+  metadata:
+    name: default
+    namespace: openshift-ingress-operator
+  spec:
+    clientTLS:
+      clientCertificatePolicy: Required
+      clientCA:
+        name: router-ca-certs-default
```

Comment:
There needs to be a step for creating the configmap specified here.  For example, the following command should work:
```
oc create configmap router-ca-certs-default --from-file=ca-bundle.pem=client-ca.crt -n openshift-config
```
It is important to note that the configmap data key needs to be `ca-bundle.pem`, and the data value needs to be a CA certificate in PEM format.